bundle/deploy/terraform/init.go

```diff
@@ -219,8 +220,12 @@ func setProxyEnvVars(ctx context.Context, environ map[string]string, b *bundle.B
 }
 
 func setUserAgentExtraEnvVar(environ map[string]string, b *bundle.Bundle) error {
-	var products []string
+	// Add databricks-dabs to the user agent with the version of the CLI. This
+	// allows us to attribute HTTP request made by the Databricks Terraform provider
+	// to DABs.
+	dabsVersion := fmt.Sprintf("databricks-dabs/%s", build.GetInfo().Version)
```

Comment:
We don't refer to bundles as `dabs` anywhere else in the code.

Could we just include the CLI version like we do in https://github.com/databricks/cli/blob/main/cmd/root/user_agent.go?

Reply:
done